engine/docker/docker.go

```diff
@@ -183,12 +185,23 @@ func (e *Docker) Destroy(ctx context.Context, pipelineConfig *spec.PipelineConfi
 
 // Run runs the pipeline step.
 func (e *Docker) Run(ctx context.Context, pipelineConfig *spec.PipelineConfig, step *spec.Step,
-	output io.Writer) (*runtime.State, error) {
+	output io.Writer, isDrone bool) (*runtime.State, error) {
 	// create the container
 	err := e.create(ctx, pipelineConfig, step, output)
 	if err != nil {
 		return nil, errors.TrimExtraInfo(err)
 	}
+
+	if !isDrone && len(step.Command) > 0 {
```

Comment:
could we move this to the caller of Run? Trying to move stuff out of engine/docker as it's pretty simple at the moment

Reply:
good suggestion will do